HPXMLtoOpenStudio/resources/data/unavailable_periods.csv

```diff
@@ -24,6 +24,7 @@ pool_heater,0,1
 permanent_spa_pump,0,1
 permanent_spa_heater,0,1
 hot_water_fixtures,1,1
+hot_water_recirculation_pump,1,1
```

Comment:
Does 0 or 1 here depend on the _type_ of recirc pump?

Reply:
Any recirculation pump requires electricity, so definitely affected by a power outage.

Reply:
Right, but what about vacancy? Should we keep it simple and just say that the recirc pump is unavailable when building is vacant?

Reply:
Seems like a reasonable default to me.